NEWS

```diff
@@ -11,9 +11,165 @@
+ * Added GDR environment variables to optimize performance on Grace Hopper systems
+ * Improved peer accessible cache for CUDA IPC to be reentrant
+#### UCS
+ * Improve non-temporal data transfer implementation
+ * Allowed configuration with wildcard prefix and name matching
+ * Defined a non-inline function to allow ASAN validation
+ * Reduced stack size in topology detection code
+ * Improved bitmaps configuration parsing with wider bitfield
+ * Added options to set topology distance between devices
+ * Optimized VFS unix socket watch by using user private folder
+ * Added general subnet matching support infrastructure
+ * Allowed user-based existing data copy when appending to array
+ * Improved time units description
+ * Added x86 non-temporal buffer transfers to skip cache
+#### UCM
+ * Added memory hooks to track allocations by CUDA memory mapping functions
+#### Tools
+ * Improved performance by increasing window size for put_bw and add get_bw in ucx_perftest
+ * Added multi-send flag for receive operations in bandwidth benchmarks in ucx_perftest
+ * Improved ucx_perftest uni-directional test with added fence
+ * Improved ucx_perftest batch helper notes
+#### Documentation
+ * Added a section regarding adaptive routing on RoCE
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+#### Build
+ * Improved compiler error reporting with added flag
+ * Improved coverity script to allow faster turnaround time
+ * Improved Intel Compiler detection and support
+#### Continuous Integration
+ * Improved spell checking on generated files
+ * Improved CI by making sure ASAN is actually loaded
+#### GO
+ * Added multi-send flag and user memh support in request params
+#### Packaging
+ * Improved dpkg-buildpackage sample command by using explicit arguments
+### Bug
+#### UCP
+ * Fixed stack overflow in exported rkey unpack
+ * Removed extra remote-cpu overhead from protocol estimation for zcopy
+ * Fixed performance estimation for rndv pipeline protocols
+ * Fixed ATP sending by picking the correct lane
+ * Fixed missing reg_id on memh creation
+ * Fixed repeated invalidations by retaining existing access flags
+ * Fixed abort reason propagation for rendezvous RTR mtype
+ * Fixed transport availability for configuration list in ALLOW mode
+ * Fixed wrong flag being used for checking BCOPY capability
+ * Fixed sending too many ATPs for small messages
+ * Enforced 16 bits size for active messages identifiers
+ * Fixed unnecessary status check for emulated AMO
+ * Fixed more than one fragment sending in rendezvous pipeline
+ * Fixed crash by using biggest max frag across all lanes
+ * Fixed missing memory handle flags by copying from parent to child
+ * Fixed worker interface activate count
+ * Fixed flush requests by replacing ATP/flush lane map with lane indexes
+ * Fixed lost uct_flags when merging memory regions
+#### UCT
+ * Fixed memory domain UCT flags description
```

Comment:
is it the same as mention in the features on line 35?
`Improved API documentation for memory domain flags`
If yes, pls remove 35-th line

Reply:
fixed